src/main/java/com/minecolonies/structures/client/TemplateRenderHandler.java

```diff
@@ -0,0 +1,61 @@
+package com.minecolonies.structures.client;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.RemovalListener;
+import com.minecolonies.blockout.Log;
+import com.minecolonies.structures.lib.TemplateUtils;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.BlockRendererDispatcher;
+import net.minecraft.client.renderer.Vector3d;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraft.util.Mirror;
+import net.minecraft.util.Rotation;
+import net.minecraft.world.gen.structure.template.Template;
+
+import java.util.concurrent.ExecutionException;
+
+import static org.lwjgl.opengl.GL11.GL_QUADS;
+
+public class TemplateRenderHandler
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this class "final" or add a public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2974)